spec/numberformat.html

```diff
@@ -457,10 +457,11 @@
       <emu-alg>
         1. Let _nf_ be *this* value.
         1. If Type(_nf_) is not Object, throw a *TypeError* exception.
-        1. If _nf_ does not have a [[boundFormat]] internal slot, throw a *TypeError* exception.
-        1. If _nf_.[[boundFormat]] is *undefined*, then
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
+        1. If _nf_.[[initializedNumberFormat]] is *undefined*, then
```

Comment:
Intl.NumberFormat and Intl.DateTimeFormat both use the internal slot name [[boundFormat]], but we need to check that the **this** value in [Intl.NumberFormat.prototype.format](https://tc39.github.io/ecma402/#sec-Intl.NumberFormat.prototype.format) is a NumberFormat instance, not a DateTimeFormat instance. Otherwise we'd violate the assertion in [NumberFormat format function, step 2](https://tc39.github.io/ecma402/#sec-number-format-functions).